native_app/pubspec.yaml

```diff
@@ -9,7 +9,7 @@ environment:
 dependencies:
   args: ^1.0.0
   build_cli_annotations: ^1.0.0
-  github: ^5.0.0
+  github: 6.0.0-dev.1.0
```

Comment:
Any particular reason to hardcode to a `version` instead of `^version`?

Reply:
Actually this package is on 6.0.2 now, fixed